requirements.txt

```diff
@@ -1,13 +1,16 @@
 pysha3
 # temporary until https://github.com/ethereum/pyethapp/pull/184 comes upstream (see also setup.py)
--e git+https://github.com/konradkonrad/pyethapp@71f940c6d287b98a35ef524f4c5e3c13d530bfc5#egg=pyethapp
+#-e git+https://github.com/konradkonrad/pyethapp@71f940c6d287b98a35ef524f4c5e3c13d530bfc5#egg=pyethapp
```

Comment:
yes it should, left it in there to ask @konradkonrad if we still need it but it's actually included in my branch anyway so it can go away